btp-document/Smart-Contracts-ICON.md

```diff
@@ -0,0 +1,387 @@
+## Smart Contracts on ICON Deployment
+
+____
+
+<p align="center">
+  <img src="./images/Deploy-Contracts-ICON.png" width="800" height="300" />
+</p>
+
+Before deploying these contracts, please make sure you have already started a local node in the ICON Network. If you have not yet done this step, please check out this [link](BTP-Development-Instructions.md#1-deploy-icon-node) and accomplish it
+
+Clone this BTP project and copy this [[Java-Folder](./files/javascore.tar)] to your local folder that has created when you started your local node
+
+```shell
+$ cd INode
+
+$ git clone https://github.com/icon-project/btp.git
+
+$ cd btp && git checkout icondao
+
+# Copy javascore.tar to $CONFIG_DIR and extract
+$ tar -xzvf javascore.tar
```

Comment:
I cannot find this file

Reply:
Javascore preparation(score developed by ICON team)
	# Copy javascore.tar to $CONFIG_DIR and extract
	$ tar -xzvf javascore.tar
		=> this one should be replaced with actual compiling way

Reply:
Nevermind, I did not notice that the file was linked a couple of lines before.